08-transport.Rmd

```diff
@@ -495,8 +552,10 @@ The output of the previous code chunk shows that `ways_sln` is a composite objec
 These include: the spatial component of the network (named `sl`), the graph component (`g`) and the 'weightfield', the edge variable used for shortest path calculation (by default segment distance).
 `ways_sln` is of class `sfNetwork`, defined by the S4 class system.
 This means that each component can be accessed using the `@` operator, which is used below to extract its graph component and process it using the **igraph** package, before plotting the results in geographic space.
+<!-- This might be a knowledge gap, but I have no idea what the edge betweeness is and what it is used for. Could you add a short explanation? -->
```

Comment:
This might be a knowledge gap, but I have no idea what the edge betweeness is and what it is used for. Could you add a short explanation?

Reply:
Discussed this today with @mem48 - yes definitely needs a brief description - thanks for flagging that. The point is treating route networks as graphs opens-up new analysis possibilities. Comments Malcolm?